src/vm/threadsuspend.cpp

```diff
@@ -6109,6 +6109,9 @@ struct ExecutionState
     IJitManager    *m_pJitManager;
     METHODTOKEN     m_MethodToken;
     BOOL            m_IsInterruptible;  // is this code interruptible?
+#if defined(_TARGET_ARM_) || defined(_TARGET_ARM64_)
+    BOOL            m_HasTailCalls;     // does this code perform tail calls?
```

Comment:
This does not need to be cached in this structure here. It is computed and used within single method - we can just call it directly right where it is needed.

Reply:
Makes sense